Package.swift

```diff
@@ -22,9 +22,9 @@ import Foundation
 var kituraNetPackage: Package.Dependency
 
 if ProcessInfo.processInfo.environment["KITURA_NIO"] != nil {
-    kituraNetPackage = .package(url: "https://github.com/IBM-Swift/Kitura-NIO.git", from: "2.0.0")
+    kituraNetPackage = .package(url: "https://github.com/IBM-Swift/Kitura-NIO.git", from: "2.1.0")
```

Comment:
These changes will need applying to the `Package@swift-4.x.swift` files too.

Reply:
I'll update them, but I don't understand why the Jenkins passed?

Reply:
In the absence of other constraints, SPM will choose the highest available release from the specified major, so `2.0.0` and `2.1.0` will both resolve to `2.1.0`.  However, if a user already had `2.0.x` (eg. in an existing build or a Package.resolved file) then that would satisfy the minimum version declared in the dependency, and the build would fail.  This won't occur in CI testing, as the build is always started from a clean state.